src/object/cli_shard.c

```diff
@@ -604,7 +605,7 @@ dc_shard_csum_report(tse_task_t *task, crt_rpc_t *rpc)
 
 	opc = opc_get(rpc->cr_opc);
 	D_ASSERT(opc == DAOS_OBJ_RPC_FETCH);
-	rc = obj_req_create(daos_task2ctx(task), &rpc->cr_ep, opc, &csum_rpc);
+	rc = obj_req_create(daos_task2ctx(task), tgt_ep, opc, &csum_rpc);
```

Comment:
the csum report always send to same target as original RPC, no? so why need tgt_ep parameter here?

Reply:
ah, you can not use cr_ep here, because daos_rpc_tag will change ep, if you use cr_ep directly, the target will be changed twice, so the RPC will send to the wrong target.

Reply:
ah, i see, thanks